### PR TITLE
Add earthworm extension

### PR DESCRIPTION
**Pull request by baboon-king**
## Description

Quick play with Earthworm.

**Earthworm** is Learning English through the method of constructing sentences with conjunctions.

[source code](https://github.com/cuixueshe/earthworm) | [home page](https://earthworm.cuixueshe.com/)

### Commands

1. `Earthworm` quick to home page
2. `Home` quick to home page
3. `Code` quick to source code page
4. `Issues` quick to issues page
5. `Pulls` quick to Pulls page

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/earthworm/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "earthworm",
+  "title": "Earthworm",
+  "description": "Quick play with Earthworm.",
```
Comment (pernielsentikaer) [inline extensions/earthworm/package.json:5]: Could be a bit more descriptive

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — COMMENTED** [review]
Everything in this extension could simply be quicklinks; perhaps it makes more sense to just use quicklinks?

**Comment by baboon-king** [comment]
> Everything in this extension could simply be quicklinks; perhaps it makes more sense to just use quicklinks?
1. 
Quicklinks is Good Idea.
But “Quicklink” can only be a single one.
It would be nice to create a collection of “Quicklink”. He should make more sense that.

2.
Then, in the near future I'm considering making the core gameplay of our site. Offered in this extension, at the moment, he really only has a few “QuickLinks”, so it's also possible to re-request a review when I'm done!

**Comment by pernielsentikaer** [comment]
It would make sense to add more since it just opens links, also right now two of the commands opens the same link 🙂

**Comment by baboon-king** [comment]
> It would make sense to add more since it just opens links, also right now two of the commands opens the same link 🙂

I'll add more since, for now let's set it up as a draft😋

**Pull request closed without merging**